feature_extract_enable : 0
point_filter_num : 2
max_iteration : 10
dense_map_enable : 1
cut_points_above_ground: false
filter_size_surf : 0.15
filter_size_map : 0.3
filter_size: 0.2
down_sample_manner: 0
cube_side_length : 20
debug : 0
img_scale : 0.5
grid_size : 40
patch_size : 8
img_enable : 1
lidar_enable : 1
equalize: true
outlier_threshold : 1200 # 
score_threshold: 100
ncc_en: false
ncc_thre: 0.0
window_size : 7
img_point_cov : 16000.0 # 1000
line_cov : 10.0 # 1000
triangulate_point_cov: 10.0
enable_triangulate: true
enable_line: true
enable_BA: false
laser_point_cov : 0.001 # 0.001
cam_fx: 453.483063
cam_fy: 453.254913
cam_cx: 318.908851
cam_cy: 234.238189
read_enable: true
pub_pcl: true
encoding: "bgr8"
map_dir: "/home/i/vio_in_lidar_map/src/vio_in_lidar_map/map/scans2.pcd"
path_file: "path2.txt"
lines_file: "line2.txt"
common:
    lid_topic:  "/livox/lidar"
    imu_topic:  "/livox/imu"

preprocess:
    lidar_type: 1 # Livox Avia LiDAR
    scan_line: 6
    blind: 5 # blind x m disable

mapping:
    acc_cov_scale: 100
    gyr_cov_scale: 10000
    acc_Q_scale: 10
    gyr_Q_scale: 10
    fov_degree:    90
    extrinsic_T: [ 0.04165, 0.02326, -0.0284 ]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]

camera:
    # img_topic: /usb_cam/image_raw
    # img_topic:  /camera/image_color
    img_topic: /left_camera/image
    #xiyuan
    Rcl: [0.00162756,-0.999991,0.00390957,
         -0.0126748,-0.00392989,-0.999912,
          0.999918,0.00157786,-0.012681]
    Pcl: [0.0409257, 0.0318424, -0.0927219]
    blind: 1
